### PR TITLE
force order of JUnit tests

### PR DESCRIPTION
**Pull request by bensonlatibm**
The com.ibm.ws.os.packaging_fat has JUnits which must run in a particular order.
Starting with Java 21, the JUnits run in a different order.

This PR collapses the JUnits into a single test method to ensure proper execution order.

### CODE REVIEW

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#run-libby-bot

**Comment by bensonlatibm** [comment]
#build

File: dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallRhelTest.java [hunk 1]
```diff
@@ -62,24 +64,16 @@ public static void cleanup() throws Exception {
...
-            ProgramOutput po = runCommand(METHOD_NAME, "rpm", param1s);
-            assertEquals("Expected exit code", 0, po.getReturnCode());
-            exiting(c, METHOD_NAME);
-        } else {
-            logger.info("OpenLiberty did not install successfully");
-        }
-    }
+        String METHOD_NAME = "testVerifyRpmInstall";
+        entering(c, METHOD_NAME);
+        assumeTrue(openLibExists);
```
Comment (jacobwdv) [inline dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallRhelTest.java]: I would recommend we pick one of these, either assume or assert. I don't think there's any way line 70 executes if `openLibExists` is false.

File: dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallRhelTest.java [hunk 2]
```diff
@@ -94,58 +88,50 @@ public void testServices() throws Exception {
 
         // service tests
         Log.info(c, METHOD_NAME, "Starting defaultServer");
-        ProgramOutput po2 = serviceCommand(METHOD_NAME, "start", "defaultServer");
+        ProgramOutput poServerStart = serviceCommand(METHOD_NAME, "start", "defaultServer");
```
Comment (jacobwdv) [inline dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallRhelTest.java:88]: much better with the new variable names 👍🏻

File: dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallUbuntuTest.java [hunk 3]
```diff
@@ -27,6 +28,7 @@
 import com.ibm.websphere.simplicity.ProgramOutput;
 import com.ibm.websphere.simplicity.log.Log;
 
+//@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```
Comment (jacobwdv) [inline dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallUbuntuTest.java]: delete this

File: dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallUbuntuTest.java [hunk 4]
```diff
@@ -61,118 +63,82 @@ public static void cleanup() throws Exception {
...
-            assertEquals("Expected exit code", 0, po.getReturnCode());
-            exiting(c, METHOD_NAME);
-        } else {
-            logger.info("OpenLiberty did not install successfully");
-        }
-
+        String METHOD_NAME = "testVerifyDebInstall";
+        entering(c, METHOD_NAME);
+
+        assumeTrue(openLibExists);
```
Comment (jacobwdv) [inline dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallUbuntuTest.java]: pick assume / assert. Don't use both

**Review by jacobwdv — CHANGES_REQUESTED** [review]
LGTM , a few nit picks.

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#build

File: dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallUbuntuTest.java (hunk 3 above)
Comment (bensonlatibm) [reply dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallUbuntuTest.java]: deleted

File: dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallUbuntuTest.java (hunk 4 above)
Comment (bensonlatibm) [reply dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallUbuntuTest.java]: changed to use assert here.

File: dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallRhelTest.java (hunk 1 above)
Comment (bensonlatibm) [reply dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallRhelTest.java]: changed to use assert.

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#libby-bot

**Review by jacobwdv — APPROVED** [review]
LGTM

**Pull request merged**